webpack/assets/javascripts/services/statisticsChartService.js

```diff
@@ -21,6 +22,37 @@ function getChartConfig(details) {
   return config;
 }
 
+function getChartConfigByData(_data, id, type) {
```

Comment:
If i understand correctly, this looks like a lot of complicated work to parse the data into the C3 format instead of sending it in the right format in the first place?